### DISCUSSION TITLE
Propogation in water?

### DISCUSSION THREAD

**Discussion by ghost** [Ifsttar/I-Simpa#351] [Q&A] [answered]
Can I-Simpa be used to simulate sound propagation in a small water chamber?

**Comment by Picaut** [comment, marked as answer]
Thank you for your question.
It seems difficult to use I-Simpa/SPPS for such configuration, although there are some similarities between the sound propagation in the air and in the water. Phenomena are the quite same, but water absorption, sound speed profile, speed of sound, reflection.. have different values/relations. In theory, I-Simpa/SPPS could be adjusted but we never  tried.
